src/Xtools/Page.php

```diff
@@ -58,6 +58,18 @@ public function getTitle()
     }
 
     /**
+     * Get the page's title without the namespace.
+     * @return string
+     */
+    public function getTitleWithoutNamespace()
+    {
+        $info = $this->getPageInfo();
+        $title = isset($info['title']) ? $info['title'] : $this->unnormalizedPageName;
+        $nsName = $this->getNamespaceName();
+        return str_replace(empty($nsName) ? '' : $nsName . ':', '', $title);
```

Comment:
I don't mind it, but I know some people are against `empty()`. In this case, it could probably check for null instead as that's what's returned by `getNamespaceName()`. Although, concatenating null with a string is a string, so maybe this can just be `str_replace($nsName . ':', '', $title);`